packages/ses/src/module-load.js

```diff
@@ -564,7 +578,11 @@ const syncJobQueue = (errors = []) => {
       try {
         func(...args);
       } catch (error) {
-        arrayPush(errors, error);
+        if (aggregateErrors) {
+          arrayPush(errors, error);
+        } else {
+          throw error;
+        }
```

Comment:
(could) if we had 2 implementations of drainQueue conditional on aggregateErrors, we could skip the try-catch entirely and make it more performant and less complex in debugging. Is it worth having more code for?